### ISSUE TITLE
Grey-out button set to false not working

### ISSUE THREAD

**Issue by potj99** [ivosturm/EditableByCondition#2]
Hi,

I have defined teh class IsEditable as the ButtonClass and assigned it to the save button. When I set the Grey-out button to Yes, the save button is greyed out when it is not editable and operational when editable. This works fine. 

When I set the grey-out button to No, the save button is never visible. Also not when the data should be editable.

Kind regards,

Jeroen

**Comment by ivosturm** [comment]
Hi Jeroen,

You are right, I just reproduced and fixed this in my local copy of the widget. If you contact me at ivo.sturm@firstconsulting.nl<mailto:ivo.sturm@firstconsulting.nl> I can send you the version with the fix in it. I will update it in the AppStoreas well, but that can take some days for Mendix to approve.

About the CK Editor, I will come back to you.

Regards,
Ivo Sturm

Van: Jeroen Pot [mailto:notifications@github.com]
Verzonden: donderdag 22 december 2016 14:29
Aan: ivosturm/EditableByCondition <EditableByCondition@noreply.github.com>
CC: Subscribed <subscribed@noreply.github.com>
Onderwerp: [ivosturm/EditableByCondition] Grey-out button set to false not working (#2)


Hi,

I have defined teh class IsEditable as the ButtonClass and assigned it to the save button. When I set the Grey-out button to Yes, the save button is greyed out when it is not editable and operational when editable. This works fine.

When I set the grey-out button to No, the save button is never visible. Also not when the data should be editable.

Kind regards,

Jereon
